alvr/session/src/settings.rs

```diff
@@ -878,6 +878,24 @@ pub enum HeadsetEmulationMode {
     },
 }
 
+#[derive(SettingsSchema, Serialize, Deserialize, PartialEq, Clone)]
+pub enum PerformanceLevel {
+    PowerSavings,
+    SustainedLow,
+    SustainedHigh,
+    Boost,
+}
+
+#[derive(SettingsSchema, Serialize, Deserialize, PartialEq, Clone)]
+pub struct PerformanceLevelConfig {
+    #[schema(flag = "real-time")]
+    #[schema(strings(display_name = "CPU"))]
+    pub cpu: PerformanceLevel,
+    #[schema(flag = "real-time")]
+    #[schema(strings(display_name = "GPU"))]
+    pub gpu: PerformanceLevel,
+}
```

Comment:
I would allow to enable overriding CPU and GPU separately. so use Switch here, and remove the Switch from `performance_level`